fission-cli/library/Fission/CLI/WebNative/FileSystem/Auth/Store.hs

```diff
@@ -1,23 +1,41 @@
 module Fission.CLI.WebNative.FileSystem.Auth.Store
-  ( getLeastPrivileged
+  ( create
+  , getLeastPrivileged
   , getMostPrivileged
   -- * Reexports
   , module Fission.CLI.WebNative.FileSystem.Auth.Store.Class
   , module Fission.CLI.WebNative.FileSystem.Auth.Store.Types
   ) where
 
 import           Crypto.Cipher.AES                                 (AES256)
+import           Crypto.Random.Types
 import qualified RIO.Map                                           as Map
 
 import           Fission.Prelude
 
 import           Fission.Error.NotFound.Types
-import qualified Fission.Key.Symmetric.Types                       as Symmetric
+import qualified Fission.Key.Symmetric                             as Symmetric
 import           Fission.User.DID.Types
 
-import           Fission.CLI.WebNative.FileSystem.Auth.Store.Class
+import qualified Fission.CLI.WebNative.FileSystem.Auth.Store.Class as WNFS.Auth
 import           Fission.CLI.WebNative.FileSystem.Auth.Store.Types
 
+-- Reexport
+
+import           Fission.CLI.WebNative.FileSystem.Auth.Store.Class
+
+create ::
+  ( MonadRandom m
+  , MonadStore  m
+  )
+  => DID
+  -> FilePath
+  -> m (Symmetric.Key AES256)
+create did path = do
+  key <- Symmetric.genAES256
+  WNFS.Auth.set did path key
+  return key
```

Comment:
We don't have a local file system, but the root key is expected by the auth lobby

Reply:
This is my biggest open question for when we have the patch on staging ☝️ Wil Drive complain?

Reply:
What do you mean by will drive complain?
What's the issue here exactly? That there's no standard directory structure initialised? I think if the WNFS isn't initialized, webnative just initializes one itself.